scripts/Get-Tags.ps1

```diff
@@ -0,0 +1,17 @@
+param(
+    [string]$Branch="master",
+    [string]$ImageBuilderImageName="microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171005132855"
+)
+
+$ErrorActionPreference = 'Stop'
+$repoRoot = Split-Path -Path "$PSScriptRoot" -Parent
+
+& docker pull $ImageBuilderImageName
+
+& docker run --rm `
+    -v /var/run/docker.sock:/var/run/docker.sock `
+    -v "${repoRoot}:/repo" `
+    -w /repo `
+    $ImageBuilderImageName `
+    generateTagsReadme `
+    "https://github.com/dotnet/dotnet-docker-nightly/blob/${Branch}"
```

Comment:
Ideally `dotnet-docker-nightly` wouldn't be hard coded so that the script can be shared between official and nightly without any changes.  This makes it easier to merge changes between the two.  This could be grabbed from the manifest - https://github.com/dotnet/dotnet-docker-nightly/blob/master/build-and-test.ps1#L21

Reply:
It would looks something like this -

```PowerShell
param(
    [string]$Branch="master",
    [string]$ImageBuilderImageName="microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171005132855",
    [string]$RepoName
)

$ErrorActionPreference = 'Stop'

$repoRoot = Split-Path -Path "$PSScriptRoot" -Parent
$remoteUrl = (git config --get remote.origin.url)
if (-not [String]::IsNullOrWhiteSpace(($remoteUrl)))
{
    $remoteUrlFragments = $remoteUrl.Split('/', [System.StringSplitOptions]::RemoveEmptyEntries)
    if ($remoteUrlFragments.Length -eq 4)
    {
        $RepoName = $remoteUrlFragments[3]
    }
}
if ([String]::IsNullOrWhiteSpace(($RepoName)))
{
    Write-Warning 'Could not automatically determine repository name. Falling back to "dotnet-docker-nightly". Add -RepoName <REPO> to override.'
    $RepoName = "dotnet-docker-nightly"
}

& docker pull $ImageBuilderImageName

& docker run --rm `
    -v /var/run/docker.sock:/var/run/docker.sock `
    -v "${repoRoot}:/repo" `
    -w /repo `
    $ImageBuilderImageName `
    generateTagsReadme "https://github.com/dotnet/${RepoName}/blob/${Branch}"
```